### PR TITLE
Testimonials section completed (v1)

### PR DESCRIPTION
**Pull request by D4MAGED**
Added Bootstrap, Reactstrap
Created Testimonials section

### CODE REVIEW

File: src/components/Testimonials/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Container, Row, Col } from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Testimonial from "../../components/Testimonial";
+
+export default class Testimonials extends Component {
+    render() {
+      return <Container><Col xs="auto">
+        <Header label="Create Something Amazing"></Header>
+        <div className="testimonial-box">
+          <Testimonial image="/images/testimonials/kyle.JPG" quotee="Keyle Atwood, attendee" body="It was great to see so many people passionate about designing interesting projects! Everyone was willing to help each other, including people from other teams, providing a great sense of community"></Testimonial>
```
Comment (kyleyarwood) [inline src/components/Testimonials/index.js]: My name is ever-so-slightly misspelt hahaha

File: src/components/Header/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        label: PropTypes.string,
+        classes: PropTypes.string
+      };
+
+    render() {
+        const { label, classes } = this.props;
+
+        return (
+          <div className={classes}>
+                      <div className="header"># {label}</div>
```
Comment (kyleyarwood) [inline src/components/Header/index.js]: Should this indentation be consistent with the previous lines?

File: src/components/Testimonials/index.js (hunk 1 above)
Comment (D4MAGED) [reply src/components/Testimonials/index.js]: Whoops, hey I'm just following the design, maybe your name needs to be changed for marketing purposes 🤣

File: src/components/Testimonial/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+class Testimonial extends Component {
+  static propTypes = {
+    image: PropTypes.string,
+    body: PropTypes.string,
+    quotee: PropTypes.string,
+  };
+
+  render() {
+    const { image, body, quotee, classes } = this.props;
```
Comment (dunja-tomic) [inline src/components/Testimonial/index.js]: Is there a reason this is quotee and not quote lol

Comment (kyleyarwood) [reply src/components/Testimonial/index.js]: I believe it's the person who was being quoted (e.g. Kyle Yarwood, haha), then the body is the actual quote

Comment (dunja-tomic) [reply src/components/Testimonial/index.js]: In that case can we change it to image, quote, and speaker/person/author/etc for clarity?

**Pull request merged**